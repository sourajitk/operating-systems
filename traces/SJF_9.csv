1,0,7
2,0,1
3,0,7
4,0,1
5,0,7
6,0,1
7,0,1
8,0,7
9,0,1
10,0,1
11,0,7
12,0,0
13,0,0
14,0,0
15,0,0
16,0,1
17,0,2
18,0,3
19,0,4
20,0,5
