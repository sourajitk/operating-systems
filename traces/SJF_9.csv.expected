1,7
2,8
3,35
4,9
5,42
6,10
7,11
8,49
9,12
10,13
11,56
12,7
13,7
14,7
15,7
16,14
17,16
18,19
19,23
20,28
